### DISCUSSION TITLE
How to customize public url file shared?

### DISCUSSION THREAD

**Discussion by felipejareta** [minio/minio#18864] [Q&A] [answered]
Hello team.

I would like to know how I can customize this link:

![image](https://github.com/minio/minio/assets/107435762/5b29d1d6-c4aa-4ac9-b51c-cc4a8bce1e20)

http://127.0.0.1:9000/etc... to http://mysite.contoso.com/etc...

I've tried some solutions as set variables but don't worked.
I am using Minio on VM and I don't have a minio.conf file.
My minio conf is in /etc/default/minio with these configurations:

MINIO_ROOT_USER="minio"
MINIO_VOLUMES="/directory/data-minio"
MINIO_OPTS="-C /etc/minio --address :9000 --console-address :9001"
MINIO_ROOT_USER=admin
MINIO_ROOT_PASSWORD="mypass"

I installed my Minio using binary steps.

Is someone can help me?

Regards.

**Comment by harshavardhana** [comment, marked as answer]
MINIO_SERVER_URL=http://host:port
